docs/user-guide/src/version_support.md

```diff
@@ -55,18 +55,21 @@ Following table summarizes BMO release/test process:
 | v0.8          | Supported |
 | v0.6          | Supported |
 | v0.5          | Tested    |
-| v0.4          | Tested    |
+| v0.4          | EOL       |
 | v0.3          | EOL       |
 | v0.2          | EOL       |
 | v0.1          | EOL       |
 
 ## Ironic-image
 
 Since `v23.1.0`, Ironic follows the semantic versioning scheme for its own
-release cycle, the same way as CAPM3 and IPAM. Currently, we have release-26.0,
-release-25.0, release-24.1, release-24.0 and release-23.1 release branches for
-`v26.0.x`, `v25.0.x`, `v24.1.x` `v24.0.x` and `v23.1.x` release cycle respectively
-and as such two or three braches are maintained as supported releases.
+release cycle, the same way as CAPM3 and IPAM.
+
+Currently, we maintain release-26.0, release-25.0, and release-24.1
```

Comment:
Can we remove this... it is not correct, and it is double maintenance compared to the table.

The table is also wrong I notice.  v24.0 is tested (as you can see from the table below the table), and v24.1 is supported (since its paired with BMO 0.6).